scenarios/setup.xml

```diff
@@ -70,10 +70,10 @@
                   exec="onos-check-apps ${OC#} org.omecproject.up4 includes"
                   requires="Up4-App-Install-Local-Build"/>
             <step name="Up4-Set-Log-Debug-${#}"
-                  exec="onos ${OC#} log:set DEBUG org.omecproject.up4"
+                  exec="${DOCKER_COMPOSE_CMD} exec -T ${OD#} apache-karaf-4.2.14/bin/client 'log:set DEBUG org.omecproject.up4'"
```

Comment:
Let's make this a command under bin to avoid duplication (and having to update the Karaf version in too many places)

Reply:
Done, I also found a way to not hardcode the Karaf version (as long as we keep `onos-test` toolset updated).

Reply:
How do we make sure `onos-test` is up-to-date. Can it happen that Karaf gets updated in sdfabric-onos but we don't get an onos-test release?

Reply:
What do you think about `$(${DOCKER_COMPOSE_CMD} exec -T "${node}" ls | grep "apache-karaf-" | sed "s/apache-karaf-//")`? In this way, we get the Karaf version directly from the container.
Long term, we could have a KARAF_VERSION env variable inside the container directly.
Please, let me know what do you think. Otherwise, we can simply hardcode the Karaf version here.